src/stores/AppStore.js

```diff
@@ -133,15 +124,10 @@ export default class AppStore extends Store {
       this.stores.router.push(data.url);
     });
 
-    const TIMEOUT = 5000;
-    // Check system idle time every minute
-    setInterval(() => {
-      this.idleTime = idleTimer.getIdleTime();
-    }, TIMEOUT);
-
```

Comment:
necessary to remove in order to avoid repeating console error `idleTimer: symbol not found`